releasenotes/notes/prelude-release-6.4.2-bfb546f51a86246c.yaml

```diff
@@ -0,0 +1,9 @@
+prelude:
```

Comment:
I think this file should be be here, the prelude for 6.4.2 is already in the changelog